### DISCUSSION TITLE
Torture: Spiral vase fractal pyramid

### DISCUSSION THREAD

**Discussion by d-schmidt** [supermerill/SuperSlicer#1968] [Q&A] [answered]
[Pyramid6](https://www.thingiverse.com/thing:1356547)

I'm looking for tipps on how to slice this. cura manages and I'm curious if it is possible to slice this in SuperSlicer.

I already disabled gap fill, top and bottom shells, increased the modell size to match 111% extrusion (on all perimeters)
[pyramid6_superslicer.zip](https://github.com/supermerill/SuperSlicer/files/7612430/pyramid6_superslicer.zip)
Slicing not only takes forever, there are some problems I can't get rid of:
without solid bottom layer it starts in the air:
![grafik](https://user-images.githubusercontent.com/3226483/143722015-7a018403-49f3-4ca4-9b58-1f90299063a0.png)
There are gaps on some layers:
![grafik](https://user-images.githubusercontent.com/3226483/143721998-67b8c039-5c81-4cb4-ae6f-d1e4983a4a71.png)

I suspect some kind of polygon correction? Something removing small details.

Cura:
![grafik](https://user-images.githubusercontent.com/3226483/143722255-200ab4ad-7de9-48d2-ab09-b597ded64aaf.png)

**Comment by neophyl** [comment, marked as answer]
I changed thew slice gap closing radius from 0.049 to 0.0049 and the first layer slices for me. Basically added a zero in there. Rest looks ok too.  (Print Settings>Slicing>Filtering>Slice Gap Closing Radius)
As for the slicing time, well its a torture test, what else do you expect ?   Took around 5 minutes to slice for me.

Btw you hadn't increased all perimeters to 111%, external perimeter was set to 105% in your project and as this is ALL external perimeter except for the skirt thats pretty much the only one you have to configure for this.  So its all printed at 0.42 except for most of the first layer.  Still slices though once you change gap closing radius.  Changing the external perimeter to 11% doesn't effect slicing though, still slices without issue.

**Reply by d-schmidt** [reply]
It was the Gap Close. Thank you
